AdyenDropIn/Presentation/DropInNavigationController.swift

```diff
@@ -58,6 +58,13 @@ internal final class DropInNavigationController: UINavigationController, Keyboar
     }
     
     internal func present(asModal component: PresentableComponent) {
+        switch component.navBarType {
```

Comment:
As mentioned [here](https://github.com/Adyen/adyen-ios/pull/551/files#r657887947) - we can get rid of this switch..